### DISCUSSION TITLE
Breaking DICOM batch 1 and 2 into single vs multi coil images

### DISCUSSION THREAD

**Discussion by jhiemstrawisc** [facebookresearch/fastMRI#259] [General]
I'm mostly interested in running learning algorithms directly on images of single coil and multi coil scans rather than on kspace data. Is there any kind of key that could allow me to divide the KNEE_DICOMS_BATCH_1 & the KNEE_DICOMS_BATCH_2 into single vs multi coil sets? Also, are these dicoms the same images that would be reconstructed from the kspace data in the other knee files, or are they a unique set of images?

Thanks!

-- Justin

**Comment by mmuckley** [comment]
Hello @jhiemstrawisc, there is no such key because it doesn't exist. We created the raw single-coil dataset mostly to help new people get acclimated to the field, but in the real world there are almost no single-coil scanners. So every image volume in the DICOM data was reconstructed from a multi-coil dataset.

They are a unique set of images in general. If I recall correctly there might be some overlap with the training set in terms of patient, but it is small and there should be no overlap with the test or challenge splits.

**Reply by jhiemstrawisc** [reply]
I'm still in the process of learning more about kspace reconstruction, but it occurs to me now that I can just reconstruct a set of single coil images from the multi-coil kspace files. In general, for the multicoil kspace data, is there any way to tell which of the coil indices from the .h5 files corresponds to posterior (ie scanning bed), anterior, body coils, etc? I'm working to see if I can reconstruct a multicoil image from just the posterior coils in the scanning bed, so if I can generate the training pairs, I should be set!

**Reply by mmuckley** [reply]
Hello @jhiemstrawisc, I don't think that information is in the .h5 files. You might try crawling through the ISMRMD header to see.
